packages/mui-material/src/styles/createPalette.d.ts

```diff
@@ -33,6 +33,7 @@ export interface TypeAction {
 export interface TypeBackground {
   default: string;
   paper: string;
+  invertChannel: string;
```

Comment:
I don't think it's safe to add this type here. The `createTheme` is not adding this value. I think we can just cast to this in the `extendTheme` util if necessary.

Reply:
Okay, sounds good to me.